kernelci/kbuild.py

```diff
@@ -38,7 +38,9 @@
 import kernelci.config.storage
 import kernelci.storage
 
-
+CIP_CONFIG_URL = \
+    "https://gitlab.com/cip-project/cip-kernel/cip-kernel-config/-\
+/raw/master/{branch}/{config}"
```

Comment:
A small nitpick: I think, it would be fine to keep this as a single longer line. The break looks a bit awkward.

Reply:
Fixed, but actually it was just copy-paste from legacy code and i guess it was like this due python linters.
But i agree, single line is cleaner, updated code.